<launch>
  <!-- The semantic description that corresponds to the URDF -->
  <param name="robot_description_semantic" textfile="$(find ur3_moveit)/config/ur3_workplace.srdf" />

  <!-- Load updated joint limits (override information from URDF) -->
  <group ns="robot_description_planning">
    <rosparam command="load" file="$(find ur3_moveit)/config/joint_limits.yaml"/>
    <rosparam command="load" file="$(find ur3_moveit)/config/paddings.yaml"/>
  </group>

  <group ns="robot_description_kinematics">
    <rosparam command="load" file="$(find ur3_moveit)/config/kinematics.yaml"/>
  </group>

</launch>
